code/modules/martial_arts/combos/sleeping_carp/gnashing_teeth.dm

```diff
@@ -0,0 +1,18 @@
+/datum/martial_combo/sleeping_carp/gnashing_teeth
+	name = "Gnashing Teeth"
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "Every second consecutive punch deals extra damage, and you will shout to strike fear into your opponent's heart."
+
+/datum/martial_combo/sleeping_carp/gnashing_teeth/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	user.do_attack_animation(target, ATTACK_EFFECT_PUNCH)
+	var/atk_verb = pick("precisely kick", "brutally chop", "cleanly hit", "viciously slam")
+	target.visible_message("<span class='danger'>[user] [atk_verb]s [target]!</span>",
+					"<span class='userdanger'>[user] [atk_verb]s you!</span>")
```

Comment:
Nitpick, but this might work better?
```suggestion
	var/atk_verb = pick("precisely kicks", "brutally chops", "cleanly hits", "viciously slams")
	target.visible_message("<span class='danger'>[user] [atk_verb] [target]!</span>",
					"<span class='userdanger'>[user] [atk_verb] you!</span>")
```